submissions/yanyshpolska/OOP-exercise/index.js

```diff
@@ -0,0 +1,78 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/Yanyshpolska/a-tiny-JS-world
+   Web app: https://yanyshpolska.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+class Inhabitant {
+  constructor(name, saying, sex) {
+    this.name = name;
+    this.saying = saying;
+    this.sex = sex;
+  }
+  getInfo() {
+    return this.name + ", " + this.saying + ", " + this.sex;
+  }
+}
+
+class Human extends Inhabitant {
+  hands = 2;
+  legs = 2;
+  constructor(...args) {
+    super(...args);
+  }
```

Comment:
`...args` is a bad practice because IDE intellisense won't be able to hint what exactly is expected here.

Reply:
Got it. 
Fixed.